packages/components/src/components/shared/FormText.tsx

```diff
@@ -130,25 +108,25 @@ export type LabelProps = FormTextProps & {
  * Label text element commonly used in form components
  */
 export const Label: FC<LabelProps> = ({ text, error, required }) => {
-  const theme = useTheme()
   const { t } = useTranslation()
 
   if (!text) return null
 
-  const textStyle = {
-    ...fontLabel,
-    fontFamily: error ? fontBold : fontRegular,
-    color: theme.vadsColorForegroundDefault,
-  }
-
-  const requiredStyle = {
-    color: theme.vadsColorForegroundError,
-  }
-
   return (
-    <Text style={textStyle}>
+    <Text
+      variant="body"
+      size="lg"
+      bottomSpacing="none"
+      bold={error ? true : false}>
       {getDisplayText(text)}
-      {required && <Text style={requiredStyle}>{` (*${t('required')})`}</Text>}
+      {required && (
+        <Text
+          variant="body"
+          size="lg"
+          tone="error"
+          bottomSpacing="none"
+          bold={error ? true : false}>{` (*${t('required')})`}</Text>
```

Comment:
Simpler if pulling out the common props here since they're the same except the tone being default on the outer and error on the inner.

Reply:
Updated to use common props